x/claim/spec/03_methods.md

```diff
@@ -6,7 +6,7 @@ order: 3
 
 ### `CompleteMission`
 
-Complete a claim mission for an eligible address.
+Complete a mission for an eligible address.
 This method can be used by an external module importing `claim` in order to define customized mission for the chain.
```

Comment:
```suggestion
This method can be used by an external module importing `claim` in order to define customized mission for the chain.
```

Should it be "external chain" or "external module"

Reply:
external chain